docs/contributing/documentation-guidelines/index.md

```diff
@@ -19,3 +21,36 @@ In terms of style, you should refer to the [Google developer documentation style
 - [Use sentence case](https://developers.google.com/style/capitalization) for document titles and section headings.
 - [Use descriptive link text](https://developers.google.com/style/link-text).
 - [Write short sentences](https://developers.google.com/style/translation#write-short,-clear,-and-precise-sentences) that are easy to understand and translate.
+
+## How to preview your modification on Autoware Documentation
+
+There are two ways to preview your modification on Autoware Documentation.
+
+### 1. Use Github Actions workflow in `autowarefoundation/autoware-documentation` repository
+
+You can use Github Actions in `autowarefoundation/autoware-documentation` repository to deploy your branch.
+
+1. Create a PR of your branch to the repository.
+2. Add a `documentation` label from the sidebar.
+3. Wait for a couple of minutes, and the `github-actions` will notify the URL for your branch's preview.
+
+### 2. Run mkdocs in your local environment
+
+Instead of creating a PR, you can use `mkdocs` to build the Autoware Documentation website on your local computer.
+Assuming that you are using Ubuntu OS, run the following to install the required libraries.
+
+```
+sudo apt install -y mkdocs
+git clone git@github.com:autowarefoundation/autoware-github-actions.git /tmp/autoware-github-actions
+cd /tmp/autoware-github-actions/deploy-docs
+pip install -U -r mkdocs-requirements.txt
+```
+
+Then, run `mkdocs serve` on your Autoware Documentation directory.
+
+```
+cd /PATH/TO/autoware-documentation
+mkdocs serve
+```
+
+It will launch mkdocs server. Access [http://127.0.0.1:8000/autoware-documentation/](http://127.0.0.1:8000/autoware-documentation/) to see the preview of Autoware Documentation.
```

Comment:
- If it's not the command, I feel using `MkDocs` is better.
- Does it require an article? (I'm not sure.)
- I think you don't need `autoware-documentation` here.

```suggestion
It will launch an MkDocs server. Access [http://127.0.0.1:8000/](http://127.0.0.1:8000/) to see the preview of Autoware Documentation.
```

Reply:
Resolved https://github.com/autowarefoundation/autoware-documentation/pull/203/commits/ff49a125b82682e0d062b355a2a4a3258451dcd9
I also changed the subsection title as well ("Running an MkDocs server in your local environment")